tests/test_hydromt.py

```diff
@@ -62,6 +62,12 @@ def test_model_build(tmpdir, model):
     # Build method options
     config = join(TESTDATADIR, _model["ini"])
     opt = parse_config(config)
+    # pop global section and assert values with init
+    global_sect = opt.pop('global')
+    assert mod1._crs == global_sect['crs']
+    assert mod1._network_snap_offset == global_sect['network_snap_offset']
+    assert (mod1._openwater_computation_node_distance == 
+            global_sect['openwater_computation_node_distance'])
```

Comment:
Is this really needed? The global options are added in lines 57-59 of this file. Maybe better to compare with the yml files of the examples rather than the test files for something really worth it?

Reply:
I have updated the branch to avoid this, I am now reading the global settings also from the yml, this makes it more robust and the test code is much simpler now (smaller `_models` dict also) and avoids the need for assertions.